### PR TITLE
README.md: add artix linux to distro list

### PR DESCRIPTION
**Pull request by kmk3**
Home page: https://artixlinux.org

A few months ago, running `pacman -S firejail` would install it from
Arch's "community" repository by default.  But currently, Artix has its
own firejail package, in the "galaxy" repository:

* https://gitea.artixlinux.org/packagesF/firejail
* https://repology.org/project/firejail/versions

See also the following article from 2021-06-09:

https://artixlinux.org/news.php#Arch_repositories_made_optional

> Arch repositories made optional
>
> Artix has reached the stage where it can operate without the help of
> the Arch repositories, including the preparation of its installation
> media.  As such, all new weekly ISO images will ship without [extra],
> [community] and [multilib] enabled in pacman.conf.  Existing setups
> will not be affected, but new users may want to enable them and
> benefit from the additional packages.  Instructions are provided in
> our wiki[1].
>
> TL;DR:
>
>     # pacman -Syu artix-archlinux-support

For reference, the distro list was added on commit ee03888ab
("prioritize installing via OS (#3442)") / PR #3442.

[1] https://wiki.artixlinux.org/Main/Repositories#Arch_repositories

Cc: @glitsj16 (as the author of #3442)

### CODE REVIEW

**Review by reinerh — APPROVED** [review]

**Comment by netblue30** [comment]
merged, thanks!
